data/battle_anim_scripts.s

```diff
@@ -15997,7 +16437,19 @@ ChillyReceptionSnowballs:
 	return
 
 gBattleAnimMove_BurningBulwark::
```

Comment:
Burning Bulawrk displays the flames on the wrong mon in doubles
![pokeemerald-test-2](https://github.com/user-attachments/assets/0b369a06-8645-449c-b397-2c4c83cc359c)